cleverpush/src/main/java/com/cleverpush/CleverPush.java

```diff
@@ -1901,6 +1925,40 @@ public void setSubscriptionCountry(String country) {
         return notifications;
     }
 
+    private void getNotificationsFromApi(NotificationFromApiCallbackListener notificationFromApiCallbackListener) {
+        String url = "/channel/" + this.channelId + "/received-notifications?";
+        ArrayList<String> subscriptionTopics = new ArrayList<String>(getSubscriptionTopics());
+
+        for (int i = 0; i < subscriptionTopics.size(); i++) {
+            if (i == 0) {
+                url = url + "topics[]=" + subscriptionTopics.get(i);
```

Comment:
This can be simplified. Initially add "?" otherwise "&".
In the next step you add topics[]=